packages/mattermost-redux/src/types/apps.ts

```diff
@@ -4,11 +4,38 @@
 // This file's contents belong to the Apps Framework feature.
 // Apps Framework feature is experimental, and the contents of this file are
 // susceptible to breaking changes without pushing the major version of this package.
+
+export enum AppType {
+    HTTP = 'http',
+    AWSLambda = 'aws_lambda',
+    Builtin = 'builtin',
+}
+
+export enum Permission {
```

Comment:
Suggest that we use the plural to match the `Locations` enum
```suggestion
export enum Permissions {
```

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html is using singular names for enums. https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN also states

> Use a singular name for most Enum types, but use a plural name for Enum types that are bit fields.


Hence, I'm leaving it as singular.